packages/client/src/impl/QueryApiImpl.ts

```diff
@@ -27,7 +27,7 @@ export default class QueryApiImpl implements QueryApi {
   private _defaultHeaders: Record<string, string> | undefined
 
   constructor(private _options: ConnectionOptions) {
-    const {host, queryTimeout, grpcOptions} = this._options
```

Comment:
I don't think we want to remove `queryTimeout` here.  It gets used as an `RpcOptions` field in the GrpcTransport constructor.